cmd/collection/main.go

```diff
@@ -94,6 +96,10 @@ func main() {
 				"set of payer addresses which are omitted from rate limiting")
 			flags.UintVar(&maxCollectionSize, "builder-max-collection-size", 200,
 				"maximum number of transactions in proposed collections")
+			flags.UintVar(&maxCollectionByteSize, "builder-max-collection-byte-size", 1000000,
+				"maximum byte size of the proposed collection")
+			flags.Uint64Var(&maxCollectionTotalGas, "builder-max-collection-total-gas", uint64(10000000),
```

Comment:
```suggestion
			flags.Uint64Var(&maxCollectionTotalGas, "builder-max-collection-total-gas", 10000000,
```

You shouldn't need type annotation for numerical literals

Reply:
Applied it.